README.md

```diff
@@ -86,22 +97,28 @@ For every script you'll find a field in the configuration file!!
 Before deploying, make sure to update the config.json with your contract type to deploy for each of the chain you wish to deploy to.
 
 Do this by running:
+
 ```bash
 # Usage: just set-contracts [chain] [contract_type]
-just set-contracts optimism MyContract
+just set-contracts optimism XCounter
```

Comment:
```suggestion
just set-contracts optimism MyContract
```
I'd still keep it MyContract so that it's clear to folks they need to write their own contracts and update them.